pkg/operator/ceph/object/zone/controller.go

```diff
@@ -139,6 +140,9 @@ func (r *ReconcileObjectZone) reconcile(request reconcile.Request) (reconcile.Re
 
 	// The CR was just created, initializing status fields
 	if cephObjectZone.Status == nil {
+		// TODO: "Created" seems confusing here and might imply to users that the object is ready
```

Comment:
I'm not a fan of the created status because under normal conditions it is going to be replaced immediately with another status such `ReconcilingStatus`. How about if we remove it?

Reply:
It seems good to initialize the status to something. My latest update renames Created to EmptyStatus and is just `""` the empty string. I think that seems good.

Reply:
What's the difference between empty status and no status? Either way, there is no status, right?

Reply:
Empty status is effectively no status, but the status struct is initialized so that we don't get nil pointer exceptions later.

Reply:
In that case, what if we initialized the struct without the API call that updates the status?